pom.xml

```diff
@@ -95,9 +95,9 @@
                 <scope>provided</scope>
             </dependency>
             <dependency>
-                <groupId>org.hibernate.validator</groupId>
-                <artifactId>hibernate-validator</artifactId>
-                <version>${version.hibernate}</version>
+                <groupId>javax.validation</groupId>
+                <artifactId>validation-api</artifactId>
+                <version>2.0.1.FINAL</version>
```

Comment:
Use Version 1.0.1.Final, which is JavaEE7, since we still require EE7

Reply:
I'm guessing we want version 1.1.0.Final? 😊

Reply:
Correct, sorry!